### PR TITLE
feat: Auth

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-4-29

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- functionality  is working fine 
- good use of navigate for routes 
- good to merge

File: play-app/src/Pages/Auth/Login/Login.jsx [hunk 1]
```diff
@@ -1,36 +1,99 @@
...
+          try {
+            const response = await axios.post("api/auth/login",{
+              email,
+           password
+            })
+            localStorage.setItem("token",response.data.encodedToken)
+            setUser(response.data.foundUser)
+            navigate(location.state?.from?.pathname ?? "/", { replace: true })
+          } catch (error) {
+            console.log(error.response)
```
Comment (ranitasahaa1230) [inline play-app/src/Pages/Auth/Login/Login.jsx:28]: Remove the console, instead show the error on the UI

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
 Looks good to merge

File: play-app/src/Pages/Auth/Login/Login.jsx (hunk 1 above)
Comment (aniketsingh13) [reply play-app/src/Pages/Auth/Login/Login.jsx:28]: will do that when add toast

**Pull request merged**